runtime/parachains/src/disputes/migration.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Storage migration(s) related to disputes pallet
+
+#[allow(deprecated)]
+use crate::disputes::{pallet::SpamSlots, Config, Pallet};
+use frame_support::{
+	pallet_prelude::*,
+	traits::{OnRuntimeUpgrade, StorageVersion},
+	weights::Weight,
+};
+/// The current storage version.
+const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
+
+pub struct MigrateToV1<T>(sp_std::marker::PhantomData<T>);
+impl<T: Config> OnRuntimeUpgrade for MigrateToV1<T> {
+	fn on_runtime_upgrade() -> Weight {
+		let mut weight: Weight = Weight::zero();
+
+		if StorageVersion::get::<Pallet<T>>() < STORAGE_VERSION {
+			weight += migrate_to_v1::<T>();
+			STORAGE_VERSION.put::<Pallet<T>>();
+		}
+		weight
+	}
+}
+
+/// Migrates the pallet storage to the most recent version, checking and setting the `StorageVersion`.
+pub fn migrate_to_v1<T: Config>() -> Weight {
```

Comment:
I need a unit test for this function.

Reply:
See `try-runtime` examples: https://github.com/paritytech/polkadot/pull/6312/files#diff-f1f55e9dfae0e13dcb140d16710bf44432f221de4469c9f6936a87bd3693d55e.

Reply:
I've added a `try-runtime` test for the disputes migration.